### ISSUE TITLE
Switch out old ammo packets for new ones

### ISSUE THREAD

**Issue by joshhartford** [api-Hypernova/AC2-cube2#52]
I noticed the sniper rifle ammo and secondary ammo packets are old models, they look like beep xD

**Comment by api-Hypernova** [comment]
Yeah I did that cuz I have no idea what they should look like LOL. Have any ideas? :D
